app/details/[ticket_id]/ticketDetails.tsx

```diff
@@ -187,20 +187,25 @@ function TicketDetails(props: Props) {
   const onChangePriority = useCallback(
     async (item: { name: string; icon: string; value: PriorityLevels }) => {
       const payload = { priority: item?.value };
```

Comment:
Instead of this just add 
if(ticketDetails?.priority !== item?.value) return;

Reply:
Done